llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -261,6 +269,21 @@ static bool funcIsSpirvSyclBuiltin(StringRef FName) {
   return FName.startswith("__spirv_") || FName.startswith("__sycl_");
 }
 
+static bool isConsideredToBeAnEntryPoint(const Function &F) {
```

Comment:
```suggestion
static bool isEntryPoint(const Function &F) {
```

Reply:
Applied in bbe37ad3deb876c5d717ac83eee7578302563c22